Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -91,11 +91,8 @@ ABenchmarkGymGameModeBase::ABenchmarkGymGameModeBase()
 	, ActorMigrationCheckDelay(5*60)
 	, PrintMetricsTimer(10)
 	, TestLifetimeTimer(0)
+	, TickActorCountTimer(0)
 	, bHasRequiredPlayersCheckFailed(false)
-	, SmoothedTotalAuthPlayers(-1.0f)
-	, SmoothedTotalAuthNPCs(-1.0f)
-	, SmoothedTotalAuthSimPlayers(-1.0f)
-	, RequiredPlayerReportTimer(10 * 60)
```

Comment:
This timer removed as we use Constants->ActorCheckDelay (which has the same delay)

Reply:
`TickActorCountTimer` ?
Looks like that has a delay of 5 mins

Reply:
Yes, a slight delay before sending ReportAuthoritativeActorCount RPCs was necessary.  Admittedly 5 minutes is probably a lot larger than needed. We only send ReportAuthoritativeActorCount if locally the actor count changes. For "stable" actors where their count does not change, only a single RPC is sent as the actor count does not change locally. I found that without a delay at the start of the deployment, the auth server was not reporting the correct counts. I do not have a explanation for this but I put it down to setting up the deployment (servers starting, game mode authority being delegated etc). I reckon the delay could be like 10 seconds and it would be fine. As long as the delay is less then 10 mins the tests should function the same. We might want earlier logging early in the deployment though (before the actual testing starts). We can do more investigation into why the delay is needed if you like. I will reduce the delay to a minute though to give us a bit more logging.